bundle/edu.gemini.pot/src/main/java/edu/gemini/pot/sp/Instrument.java

```diff
@@ -0,0 +1,62 @@
+package edu.gemini.pot.sp;
+
+import static edu.gemini.pot.sp.SPComponentType.*;
+import edu.gemini.shared.util.immutable.ImOption;
+import edu.gemini.shared.util.immutable.Option;
+import edu.gemini.shared.util.immutable.None;
+import edu.gemini.shared.util.immutable.Some;
+
+import java.util.Arrays;
+
+/**
+ * An enumeration of all instruments, which is a subset of all component types.
+ */
+public enum Instrument {
+    AcquisitionCamera(INSTRUMENT_ACQCAM),
+    Bhros(INSTRUMENT_BHROS),
+    Flamingos2(INSTRUMENT_FLAMINGOS2),
+    GmosNorth(INSTRUMENT_GMOS),
+    GmosSouth(INSTRUMENT_GMOSSOUTH),
+    Gnirs(INSTRUMENT_GNIRS),
```

Comment:
I guess wewill have to add `Ghost` here. I have been trying to mark such areas of the code as ``TODO:GHOST` so that we can grep through and not forget.

Reply:
I'll add that here and add a completeness test case as well.